src/external/CMakeLists.txt

```diff
@@ -23,14 +23,17 @@ endif()
 
 option(CAPSTONE_BUILD_SHARED "Build shared library" OFF)
 option(CAPSTONE_BUILD_TESTS "Build tests" OFF)
-option(CAPSTONE_ARM_SUPPORT "ARM support" ON)
```

Comment:
With remove this line if you want on it ?

Reply:
Because the default value is ON

Reply:
See https://github.com/aquynh/capstone/blob/master/CMakeLists.txt#L40

Reply:
because it's on by default
I agree thought, shouldn't have been removed probably

Reply:
yes i know, just me i leave it here ^^